_authors/osci_2023/rileyli.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Riley Li
+short_name: Riley
+twitter:
```

Comment:
@MadaniKK Remove this line instead of leaving blank.

Reply:
Done! sorry I didn't see the comment till now! Thank you for reviewing my PR!